Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -101,12 +124,27 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
     self.configureWithDataProperty.value = data
   }
 
+  private let (goToLoginSignupSignal, goToLoginSignupObserver) = Signal<Void, Never>.pipe()
```

Comment:
First time I've seen this, what's the goal of using `Signal<Void, Never>.pipe()`?

Reply:
Hmm, to be honest, this is also just copy-pasted from the standard pledge version of this VC/VM. But I think the point is that we want to react to button clicks - so we want to track whenever new events happen but we don't actually need it to pass along any data.

Reply:
Aah, gotcha. I think we mix `Void` and `()` throughout the codebase. Seems fine.